static/css/add.scss

```diff
@@ -0,0 +1,266 @@
+      padding-left: 0.1em;
+      margin-bottom: 0.5em;
+    }
+
+    input {
+
+    }
+  }
+
+  // Styling dropdown menu
+  &__category {
+    overflow: hidden;
+
+    select {
+      display: block;
+      font-family: $title;
+      font-size: 1em;
+      font-weight: 800;
+      text-transform: uppercase;
+      color: $secondary;
+      background-color: white;
+      border: 0.05em solid $main;
+      border-radius: 0.5em;
+      padding: 0.5em 0.5em;
+      margin-bottom: 0.5em;
+      transition: box-shadow 0.3s;
+
+      // Changing the dropdown arrow to a chevron
+    	background: url("http://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/br_down.png") right center no-repeat;
+    	appearance: none;
+
+      &:hover {
+        box-shadow: 0 0 0.3em $secondary;
+      }
+
+      option {
+        font-family: $title;
+        font-size: 1em;
+        font-weight: 800;
+        text-transform: uppercase;
+      }
+
+      option[value="academic_calendar"] {
+        color: $academic;
+      }
+
+      option[value="academic_advising"] {
+        color: $academic;
+      }
+
+      option[value="residential"] {
+        color: $student;
+      }
+
+      option[value="international"] {
+        color: $student;
+      }
+
+      option[value="diversity"] {
+        color: $student;
+      }
+
+      option[value="pgp"] {
+        color: $student;
+      }
+
+      option[value="hr"] {
+        color: $student;
+      }
+
+      option[value="health"] {
+        color: $student;
+      }
+
+      option[value="admission"] {
+        color: $admission;
+      }
+
+      option[value="library"] {
+        color: $library;
+      }
+
+      option[value="shop"] {
+        color: $shop;
+      }
+
+      option[value="clubs"] {
+        color: $clubs;
+      }
+
+      option[value="other"] {
+        color: $other;
+      }
+    }
+  }
+
+  // Row for each form line
+  &__field {
+    display: block;
+
+    &--flex {
+      display:flex;
+      justify-content: flex-start;
+    }
+
+    label {
+      display: block;
+      text-align: left;
+      font-family: $title;
+      font-size: 1em;
+      font-weight: 800;
+      color: $form-header;
+      text-transform: uppercase;
+      padding-left: 0.1em;
+      margin-bottom: 0.5em;
+    }
+
+    input {
+      display: block;
+      text-align: left;
+      width: 100%;
+      border: none;
+      border-radius: 0.5em;
+      background-color: $secondary-light;
+      padding: 0.5em;
+      margin-bottom: 0.5em;
+      font-family: $sans;
+      font-size: 1em;
+      color: $main;
+
+      &::placeholder {
+        color: $secondary;
+      }
+    }
+
+    textarea {
+      display: block;
+      text-align: left;
+      width: 29.5em;
+      height: 7em;
+      border: none;
+      border-radius: 0.5em;
+      background-color: $secondary-light;
+      padding: 0.75em 0.5em;
+      margin-bottom: 0.5em;
+      font-family: $sans;
+      font-size: 1em;
+      color: $main;
```

Comment:
Add `resize: vertical` to prevent horizontal resizing.

[See here.](https://stackoverflow.com/questions/25567707/textarea-disable-resize-on-x-or-y)